content/docs/components/serving/istio.md

```diff
@@ -11,21 +11,26 @@ quota, rollout and A/B testing.
 We assume Kubeflow is already deployed in the `kubeflow` namespace.
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/install/crds.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/install/istio-noauth.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/kf-istio-resources.yaml
+$ curl -L https://git.io/getLatestIstio | ISTIO_VERSION=1.3.3 sh -
```

Comment:
Hey @capt2101akash by default, the isito will be installed by kubeflow. Could you please mention here that if the istio is not installed, you can ...

Reply:
sure will do